kafka/consumer/fetcher.py

```diff
@@ -696,12 +703,21 @@ def _create_fetch_requests(self):
                 #       dicts retain insert order.
                 partition_data = list(partition_data.items())
                 random.shuffle(partition_data)
-                requests[node_id] = FetchRequest[version](
-                    -1,  # replica_id
-                    self.config['fetch_max_wait_ms'],
-                    self.config['fetch_min_bytes'],
-                    self.config['fetch_max_bytes'],
-                    partition_data)
+                if version == 3:
```

Comment:
I think this should be `<= 3` ?

Reply:
no, the 0,1 and 2 versions are handled above

Reply:
yep -- i missed that!